cabal.project

```diff
@@ -26,6 +26,11 @@ source-repository-package
     location: https://github.com/nuttycom/true-name.git
     tag: 6ec32f4170a4bf823f6d80d1e3b6dc3e18746d87
 
+source-repository-package
+    type: git
+    location: https://github.com/nuttycom/lrzhs.git
+    tag: b10c8cc245f2353dfe3f5cbd6e231082a23ced7d
```

Comment:
Is it possible to tag a...git tag? I know maybe you don't want HEAD, but if there were a "stable" tag would that help with ensuring up-to-date dependencies? Alternatively, are there tools that help you audit when dependencies are behind? Same goes for the tags in the flake files

Reply:
I think that it's good to use tags when you actually stabilize things, but I expect these versions (particularly this one) to be evolving pretty quickly. With respect to the `thyme` and `true-name` dependencies, those are more stable but I keep hoping that their maintainer will upstream my patches so that I can get rid of those entirely. It's not clear they've even seen them though, despite that I've made a number of attempts to contact them.